frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx

```diff
@@ -71,13 +71,26 @@ const filterSelected = (list: BackingStoreStateType[], tableId: number) => {
   return list.filter((e) => e.selectedBy === tableId || e.selectedBy === '').sort(sort);
 };
 
-const getTableRows = (list: K8sResourceKind[]) => {
+const isSelectedBy = (bs: K8sResourceKind, tier1: string[], tier2: string[]): number => {
+  if (tier1.length) {
+    const searchTier1 = tier1.find((elem) => elem === bs.metadata.name);
+    if (searchTier1) return 0;
+  } else if (tier2.length) {
+    const searchTier2 = tier2.find((elem) => elem === bs.metadata.name);
```

Comment:
You should create a single function for ` tier2.find((elem) => elem === bs.metadata.name);` since its used at 2 places.

Reply:
Could you suggest a snippet or something? I am not getting your suggestion.

Reply:
`const  findInTier = (tier, name) => tier.find((elem) => elem === name)`
and call it like 
`  const searchTier1 = findInTier(tier1, bs.metadata.name)`
`  const searchTier2 = findInTier(tier2, bs.metadata.name)`

Reply:
ack